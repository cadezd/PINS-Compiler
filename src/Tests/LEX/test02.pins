0 00 000
1 11 111
2 22 222
1 01 001
2 02 020

12345 67890
10001001010
00000110001