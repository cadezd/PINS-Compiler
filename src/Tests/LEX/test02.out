[1:1] C_INTEGER:0
[1:3] C_INTEGER:0
[1:4] C_INTEGER:0
[1:6] C_INTEGER:0
[1:7] C_INTEGER:0
[1:8] C_INTEGER:0
[2:1] C_INTEGER:1
[2:3-2:4] C_INTEGER:11
[2:6-2:8] C_INTEGER:111
[3:1] C_INTEGER:2
[3:3-3:4] C_INTEGER:22
[3:6-3:8] C_INTEGER:222
[4:1] C_INTEGER:1
[4:3] C_INTEGER:0
[4:4] C_INTEGER:1
[4:6] C_INTEGER:0
[4:7] C_INTEGER:0
[4:8] C_INTEGER:1
[5:1] C_INTEGER:2
[5:3] C_INTEGER:0
[5:4] C_INTEGER:2
[5:6] C_INTEGER:0
[5:7-5:8] C_INTEGER:20
[7:1-7:5] C_INTEGER:12345
[7:7-7:11] C_INTEGER:67890
[8:1-8:11] C_INTEGER:10001001010
[9:1] C_INTEGER:0
[9:2] C_INTEGER:0
[9:3] C_INTEGER:0
[9:4] C_INTEGER:0
[9:5] C_INTEGER:0
[9:6-9:11] C_INTEGER:110001
EOF:EOF